server/utils/checkYourAnswersUtils/index.ts

```diff
@@ -47,7 +47,7 @@ const summaryListItemForResponse = (
         href: paths.applications.pages.show({ task: task.id, page: pageName, id: application.id }),
         text: 'Change',
         visuallyHiddenText: key,
-        classes: 'exclude-from-print',
+        classes: 'govuk-\\!-display-none-print',
```

Comment:
Those backslashes (escaping?) shouldn't be needed; they result in the class name being rendered as `govuk-\!-display-none-print`, which means the links aren't actually hidden when trying to print!

Reply:
Should be fixed now